modules/network/traffic-manager/test/main.test.bicep

```diff
@@ -10,23 +11,34 @@ module file is a deployment test. Make sure at least one test is added.
 //     location: location
 //   }
 // }
+param prefix string = 'traf'
+@maxLength(60)
+param name string = take('${prefix}-${uniqueString(resourceGroup().id, subscription().id)}', 60)
 
 // Test 0
 module test0 '../main.bicep' = {
   name: 'test0'
+  params: {
+    name: 't0${name}'
+    tags: {
+      env: 'test'
+    }
+    trafficManagerDnsName: 'tmp0-${uniqueString(resourceGroup().id, subscription().id, name)}'
+  }
 }
 
 // Test 1
 // Set Traffic Manager Endpoint
 module test1 '../main.bicep' = {
   name: 'test1'
   params: {
-    prefix: 'traf1'
-    endpoints: [{
+    name: 't1${name}'
+    trafficManagerDnsName: 'tmp1-${uniqueString(resourceGroup().id, subscription().id, name)}'
+    endpoints: [ {
         name: 'my-endpoint-1'
         target: 'www.bing.com'
         endpointStatus: 'Enabled'
         endpointLocation: 'eastus'
-    }]
+      } ]
```

Comment:
nit: Formatting.

Reply:
I think it's bicep fmt did

Reply:
Ah, don't worry about it then. That's a weird Bicep formatter behavior, though. I'm going to fix it later.

Reply:
Do we need to raise bug for it ?